README.md

```diff
@@ -154,175 +154,176 @@ If you are interested, please contact us at zigrazor@gmail.com or contribute to
 
 ### Install Linux Tarballs
 
-On Unix/Linux system you need to execute the following command to install:
+To install on Unix/Linux systems, execute the following from the command line:
 
 `$ sudo tar xjf CXXGraph-{version}.tar.bz2`
 
-to uninstall:
+To uninstall:
 
 `$ sudo rm -f /usr/include/Graph.hpp /usr/include/CXXGraph*`
 
 ### Install RPM
 
-On Fedora/CentOS/RedHat system you need to execute the following command to install:
+To install on Fedora/CentOS/RedHat systems, execute the following from the command line:
 
 `$ sudo rpm -ivh CXXGraph-{version}.noarch.rpm`
 
-to uninstall:
+To uninstall:
 
 `$ sudo rpm -e CXXGraph-{version}`
 
 ### Install DEB
 
-On Debian/Ubuntu system you need to execute the following command to install:
+To install on Debian/Ubuntu systems, execute the following from the command line:
 
 `$ sudo dpkg -i CXXGraph_{version}.deb`
 
-to uninstall:
+To uninstall:
 
 `$ sudo apt-get remove CXXGraph`
 
 ### Install From Source
 
-You can install from source the library using CMake. After the compilation phase, you can use:
+For self-compiled installations using CMake, execute the following from the command line once compilation is complete:
 
 `$ sudo make install`
 
-to install the library.
-
 ## Classes Explanation
 
-The Classes Explanation can be found in the [Doxygen Documentation](https://rawcdn.githack.com/ZigRazor/CXXGraph/master/docs/html/index.html), in the [Classes Section](https://rawcdn.githack.com/ZigRazor/CXXGraph/master/docs/html/classes.html)
+The Classes Explanation can be found in the [Classes Section](https://rawcdn.githack.com/ZigRazor/CXXGraph/master/docs/html/classes.html) of the [Doxygen Documentation](https://rawcdn.githack.com/ZigRazor/CXXGraph/master/docs/html/index.html)
 
-## Requirements
+## Prerequisites
 
-- The minimum C++ standard required is **C++17**
-- A GCC compiler version greater than 7.3.0 *OR*
-- A MSVC compiler that supports C++17
+- The minimum C++ standard required is **C++17** 
+- A GCC compiler version 7.3.0 and later *OR* a MSVC compiler that supports C++17
 
 ## How to use
 
-The use of the library is very simple, **just put the header file where you need!**
+To use the library **simply put the header file where you need it.** It's that easy!
```

Comment:
I don't think this is strictly true yet. I think there were plans to add a script to merge all the code into a single header. Do we have an issue to track this @ZigRazor ?

Reply:
not yet, but it can be open!